5-Heroku/server/middlewares/checkCookie.js

```diff
@@ -0,0 +1,14 @@
+
+const {v4: uuidv4} = require('uuid');
+
+const checkCookie = function (req, res, next) {
+    const userId = req.cookies['user_id'];
+    if (!userId) {
+        const userId = uuidv4();
+        res.cookie('user_id', userId, {httpOnly: true, secure: false });
```

Comment:
what are secure / httpsOnly flags and why did u set secure to false?

Reply:
it was there when i looked on how to get cookies.
I can delete them both for now, but I assume that using https is safer.
Regarding the secure - my bad and it better be true to make sure the cookie is sent when the request is https (or localhost).

Should i remove both?